### DISCUSSION TITLE
[v3.5.2] Alias Suggestions

### DISCUSSION THREAD

**Discussion by KaguraSayuki** [Arcaea-Infinity/BotArcAPIs-Memories#23] [Song Alias] [answered]
New alias requests for new song:

GIMME DA BLOOD (_gimmedablood_) - 大菠萝
GIMME DA BLOOD (_gimmedablood_) - 大出血
GIMME DA BLOOD (_gimmedablood_) - 献血
GIMME DA BLOOD (_gimmedablood_) - gdb

**Comment by Awbugl** [comment, marked as answer]
gou => 业善
gou => 业
